RELEASING.md

```diff
@@ -0,0 +1,204 @@
+# Release Documentation
+
+This document describes the process on how to release a new version of Valkey-GLIDE, how to use the GLIDE's CD workflows, and how to release a new wrapper.
+
```

Comment:
On the top, we should include a "Release Schedule" section.  We can leave it as a "this section is under review", but the schedule is something we should publish somewhere and link to it here. 

The schedule should probably be on the wiki.